### ISSUE TITLE
Aborted connection # to db: 'unconnected' user: 'unauthenticated' host: 'x.x.x.x' (This connection closed normally without authentication)

### ISSUE THREAD

**Issue by sastromo** [wallabag/docker#420]
Hello
Since omnivore is going down, I decided to try wallabag, but I hit a snag.

I'm trying to install via docker compose.
Everything gets installed
![Screenshot_67](https://github.com/user-attachments/assets/2a8d366c-a531-478e-888f-3d76b0ddda6a)

But...
Mariadb returns: 
`[Warning] Aborted connection 3 to db: 'unconnected' user: 'unauthenticated' host: '172.29.0.4' (This connection closed normally without authentication)`

I found the supposed solution to this issue at the [mariadb-docker](https://github.com/MariaDB/mariadb-docker/issues/560) github page: 
Adding this 
```command: --character-set-server=utf8mb4 --collation-server=utf8mb4_bin```
to the mariadb compose would fix the problem.. 

So I did and nothing changed.. it still returns the same error.

I already tried:
- several different mariadb versions from 10.4 to 11.4.2.
- only `--character-set-server=utf8mb4` in command

Have anyone gone through the same problem?
Thank you!

**Comment by mpcrossley** [comment]
Yes! Same problem here...

**Comment by paulr24** [comment]
Same

**Comment by euinor** [comment]
I had the same error, no matter which database type I used (tried sqlite, mariadb and postgres).

This reddit comment helped me:
https://www.reddit.com/r/selfhosted/comments/qbig4g/comment/hsi41zk/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button

The same commands are also referenced here:
https://github.com/wallabag/docker/pull/392

TL;DR
Add `POPULATE_DATABASE=false` to your compose file (under Wallabag service)
In terminal run the following:
`docker container ls -a`
Take the ID of wallabag and use it in these commands
```
$ docker exec -t <ID> /var/www/wallabag/bin/console wallabag:install --env=prod --no-interaction
$ docker exec -t <ID> /var/www/wallabag/bin/console doctrine:migrations:migrate --env=prod --no-interaction
```
